Sprint-3/reading-list/script.js

```diff
@@ -1,3 +1,28 @@
+function readingList(books) {
+  const readingListContainer = document.getElementById("reading-list");
+
+  books.forEach((book) => {
+    const listItem = document.createElement("li");
+    listItem.classList.add("book-item", book.alreadyRead ? "green" : "red");
+
+    const titleAuthor = document.createElement("h3");
+     titleAuthor.innerHTML = `<span style="font-size: 2rem;">${book.title}<span>
+     <span style="font-size: 1.5rem;">by </span>
+     <span style="font-size: 2rem;">${book.author}</span>`;
+    listItem.appendChild(titleAuthor);
+
+    const coverImage = document.createElement("img");
```

Comment:
dont forget to add alt tgs for all your images. This is in case thew image doesn't load correctly or for someone who is blind to know what the image is supposed to show